pwa/src/pages/endpoints/[endpointId].tsx

```diff
@@ -26,15 +26,7 @@ const IndexPage = (props) => {
                   }
                 ]}
               />
-            ) : (
-              <Tabs
-                items={[{ name: "Overview", id: "overview", active: true },
-                  {
-                    name: "Logs",
-                    id: "logs"
-                  }]}
-              />
-            )}
+            )
```

Comment:
Something's going wrong here. The title shows as "Logs" which should be "Endpoint". Also, there's some code showing in the frontend (probably some issues that were already there, but maybe you could fix it in this PR 👍🏼 ).

Edit: and the Log table shouldn't show underneath the form (it should only show when the tab is active).

![image](https://user-images.githubusercontent.com/66552932/155296804-24e13245-8e21-4854-852f-4fcdcb6464e5.png)